yarn/src/main/scala/org/apache/spark/deploy/yarn/ApplicationMaster.scala

```diff
@@ -430,6 +430,10 @@ private[spark] class ApplicationMaster(args: ApplicationMasterArguments,
   private def startUserClass(): Thread = {
     logInfo("Starting the user JAR in a separate Thread")
```

Comment:
no, what I mean is it will become outdated with your changes, since we won't be "starting the user JAR" if we're running a python application